packages/office-ui-fabric-react/src/components/Facepile/examples/Facepile.Basic.Example.tsx

```diff
@@ -100,4 +106,16 @@ export class FacepileBasicExample extends React.Component<{}, IFacepileBasicExam
       }
     );
   };
+
+  private _personaPresence = (personaName: any): any => {
```

Comment:
Does this have to use any? As far as I can tell you should be able to do `personaName: string` and `PersonaPresence` for the return type and array type (with `!` operators in places if needed).

Reply:
You got it!